libqtile/resources/default_config.py

```diff
@@ -151,8 +151,8 @@
 mouse = [
     Drag([mod], "Button1", lazy.window.set_position_floating(),
          start=lazy.window.get_position()),
-    Drag([mod], "Button3", lazy.window.set_size_floating(),
-         start=lazy.window.get_size()),
+    Drag([mod], "Button3", lazy.window.drag_resize_floating(),
+         start=lazy.window.start_resize()),
```

Comment:
I don't see a point in keeping both implementations alive, since this one is strictly superior in terms of user experience. I think we should modify `set_position_floating` and `get_position` directly (and whatever mechanism used in `Drag`) rather than implementing new methods.

Reply:
Hopefully this isn't easier said than done. I'll dive right in!